samples/wifi/raw_tx_packet/Kconfig

```diff
@@ -81,4 +81,38 @@ config RAW_TX_PACKET_SAMPLE_CHANNEL
 	  Specify the Wi-Fi channel for transmitting raw TX packets in non-connected Station mode.
 	  The valid channel range is 1 to 233.
 
+config RAW_TX_PACKET_SAMPLE_RATE_VALUE
+	int "MCS index or Data rate value"
+	default 0
+	help
+	  Specify the Wi-Fi data rate value for raw TX packet transmission.
+	  The value represents the data rate in Mbps.
```

Comment:
```
The value represents either the data rate in Mbps or Modulating and coding (MCS) index.
Note: 5.5Mbps is treated specially as 55 to avoid floating point usage.
```